From 01c942744635484d91ed70a1cb23f492ebf30f2b Mon Sep 17 00:00:00 2001
From: Sandy Ryza <sandy@cloudera.com>
Date: Thu, 14 Mar 2013 13:15:03 -0700
Subject: [PATCH 1295/1357] MR1: MAPREDUCE-4777. In TestIFile, testIFileReaderWithCodec relies on testIFileWriterWithCodec. Contributed by Sandy Ryza

Reason: Fix tests broken by move to JDK7
Ref: CDH-10994
Author: Sandy Ryza
---
 .../test/org/apache/hadoop/mapred/TestIFile.java   |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/hadoop-mapreduce1-project/src/test/org/apache/hadoop/mapred/TestIFile.java b/hadoop-mapreduce1-project/src/test/org/apache/hadoop/mapred/TestIFile.java
index 0411711..ee75e9b 100644
--- a/hadoop-mapreduce1-project/src/test/org/apache/hadoop/mapred/TestIFile.java
+++ b/hadoop-mapreduce1-project/src/test/org/apache/hadoop/mapred/TestIFile.java
@@ -56,6 +56,10 @@ public class TestIFile {
     Path path = new Path(new Path("build/test.ifile"), "data");
     DefaultCodec codec = new GzipCodec();
     codec.setConf(conf);
+    IFile.Writer<Text, Text> writer =
+      new IFile.Writer<Text, Text>(conf, rfs, path, Text.class, Text.class,
+                                   codec, null);
+    writer.close();
     IFile.Reader<Text, Text> reader =
       new IFile.Reader<Text, Text>(conf, rfs, path, codec, null);
     reader.close();
-- 
1.7.0.4

